kustomize-fns/v1alpha1/applyresources/example/opendatahub.yaml

```diff
@@ -0,0 +1,179 @@
+          name: manifests
+          path: pytorch-job/pytorch-job-crds
+      name: pytorch-job-crds
+    - kustomizeConfig:
+        overlays:
+          - application
+        repoRef:
+          name: manifests
+          path: pytorch-job/pytorch-operator
+      name: pytorch-operator
+    - kustomizeConfig:
+        overlays:
+          - application
+        parameters:
+          - name: usageId
+            value: <randomly-generated-id>
+          - name: reportUsage
+            value: 'true'
+        repoRef:
+          name: manifests
+          path: common/spartakus
+      name: spartakus
+    - kustomizeConfig:
+        overlays:
+          - application
+        repoRef:
+          name: manifests
+          path: tf-training/tf-job-crds
+      name: tf-job-crds
+    - kustomizeConfig:
+        overlays:
+          - application
+        repoRef:
+          name: manifests
+          path: tf-training/tf-job-operator
+      name: tf-job-operator
+    - kustomizeConfig:
+        overlays:
+          - application
+        repoRef:
+          name: manifests
+          path: katib/katib-crds
+      name: katib-crds
+    - kustomizeConfig:
+        overlays:
+          - application
+        repoRef:
+          name: manifests
+          path: pipeline/api-service
+      name: api-service
+    - kustomizeConfig:
+        overlays:
+          - application
+        parameters:
+          - name: minioPvcName
+            value: minio-pv-claim
+        repoRef:
+          name: manifests
+          path: pipeline/minio
+      name: minio
+    - kustomizeConfig:
+        overlays:
+          - application
+        parameters:
+          - name: mysqlPvcName
+            value: mysql-pv-claim
+        repoRef:
+          name: manifests
+          path: pipeline/mysql
+      name: mysql
+    - kustomizeConfig:
+        overlays:
+          - application
+        repoRef:
+          name: manifests
+          path: pipeline/persistent-agent
+      name: persistent-agent
+    - kustomizeConfig:
+        overlays:
+          - application
+        repoRef:
+          name: manifests
+          path: pipeline/pipelines-runner
+      name: pipelines-runner
+    - kustomizeConfig:
+        overlays:
+          - application
+        repoRef:
+          name: manifests
+          path: pipeline/pipelines-viewer
+      name: pipelines-viewer
+    - kustomizeConfig:
+        overlays:
+          - application
+        repoRef:
+          name: manifests
+          path: pipeline/scheduledworkflow
+      name: scheduledworkflow
+    - kustomizeConfig:
+        overlays:
+          - application
+        repoRef:
+          name: manifests
+          path: pipeline/pipeline-visualization-service
+      name: pipeline-visualization-service
+# Add transformer config to manifests repo
+  global:
+    transformers:
+      - name: Apply Resources
+        repoRef:
+          name: manifests
+          path: transformers/resourceApply.yaml
+  repos:
```

Comment:
I am not clear what this specifically does. Are we suggesting we change the Kfdef format to allow for globals:?

Reply:
This is required to add custom transformer. In our case its `resourceApply`. The file has the config required for the plugin